data/json/items/comestibles/med.json

```diff
@@ -1337,5 +1337,24 @@
             "move_cost" : 300,
             "used_up_item" : { "id": "cotton_ball", "quantity": 1, "charges": 1, "flags": [ "FILTHY" ] }
         }
+    },
+    {
+        "id" : "amoxicillin",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "amoxicillin",
+        "name_plural" : "amoxicillin",
+        "description" : "A broad-spectrum antibiotic used to suppress infections and prevent them from setting in. It isn't strong enough to purge infections outright, but it boosts the body's resistance against them.",
```

Comment:
Add dosage directions to the description, "Take one dose every 12 hours until symptoms cease".